scripts/macos_build_universal.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env zsh
```

Comment:
Does it have to be zsh? That's probably not installed on CI by default, is it?

Would probably also need to sprinkle doublequotes around all the variable expansions to handle spaces in paths etc. Not sure if zsh needs it; bash probably does.

Reply:
Also probably better add `set -e` at the top of the script to stop on any error.

Reply:
macOS ships with `zsh` as the standard shell, and since this will (most likely, usually) be run on a Mac system I opted for `zsh`. But I also don't really use many of the fancier features in it, so I can have it set to use whatever the system shell is instead.